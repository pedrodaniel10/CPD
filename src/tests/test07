3 5 500 5