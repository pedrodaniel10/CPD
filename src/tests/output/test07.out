0.60 0.27
0.49 0.49
